apps/src/codebridge/Codebridge.tsx

```diff
@@ -147,6 +149,11 @@ export const Codebridge = React.memo(
       config.layoutComponents,
     ]);
 
+    const backpackApi = useMemo(
+      () => new BackpackClientApi('pythonlab', null),
```

Comment:
We shouldn't hard-code 'pythonlab' here since this is also used for web lab 2. We could get the app type from redux.

Reply:
Thanks - updated!